packages/common/src/utils/formatUtil.ts

```diff
@@ -34,6 +34,36 @@ export const formatCount = (count: number) => {
   }
 }
 
+/**
+ * The format for USDC should be:
+ * - show 0 if 0
+ * - don't show decimal places if input is a round number
+ * - show only up to 2 decimal places if input is not a round number
+ * - round to nearest thousand if input is greater than 10000
```

Comment:
This seems wrong?

Reply:
what else is wrong apart from rounding down instead of rounding to nearest thousand?

Reply:
Just that! Sorry, saw things out of order